frontend/appflowy_flutter/lib/plugins/database_view/application/filter/filter_service.dart

```diff
@@ -84,10 +84,17 @@ class FilterBackendService {
     required String fieldId,
     String? filterId,
     required DateFilterConditionPB condition,
+    required FieldType fieldType,
     int? start,
     int? end,
     int? timestamp,
   }) {
+    assert(
```

Comment:
Not sure if it's possible in asserts, I normally don't use them that much in constructors tbf.

But another way of writing this conditional would be:

`[FieldType.DateTime, FieldType.UpdatedAt, FieldType.CreatedAt].contains(fieldType)`

Reply:
Could you add a trailing comma after `FieldType.CreatedAt`? It's my personal opinion that it will improve the readability. +1 to this suggestion @Xazin.